src/app/containers/Figure/index.test.jsx

```diff
@@ -36,6 +36,27 @@ describe('Figure', () => {
     expect(Object.keys(wrapper.props()).length).toBe(7);
   });
 
+  it('should load lazyload component and transition component when lazyLoad prop is set to true', () => {
```

Comment:
Shouldn't you also have a failing case?

Reply:
Thanks, @oluoluoxenfree, I think that case should be added as well